app/components/TumourSummaryEdit/index.tsx

```diff
@@ -323,32 +339,27 @@ const TumourSummaryEdit = ({
     return null;
   }, [handleClicked, handleDelete, handleKeyDown, newMicrobialData]);
 
-  const mutBurDataSection = useMemo(() => {
-    if (newMutationBurdenData) {
-      return (
-        <TextField
-          className="tumour-dialog__text-field"
-          label="Mutation Burden (Mut/Mb)"
-          value={newMutationBurdenData.totalMutationsPerMb}
-          name="totalMutationsPerMb"
-          onChange={handleMutationBurdenChange}
-          variant="outlined"
-          fullWidth
-          type="number"
-        />
-      );
-    }
-    return null;
-  }, [newMutationBurdenData, handleMutationBurdenChange]);
+  const mutBurDataSection = (
+    <TextField
+      className="tumour-dialog__text-field"
+      label="Mutation Burden (Mut/Mb)"
+      value={newMutationBurdenData?.totalMutationsPerMb ?? 0}
```

Comment:
Should we change the default value of the field from '0' to 'null'? I know it won't save without a specific event that triggers the onChange but just incase the ClinInfo analysts see the value of 0 and wonder if it's 0 in the DB

Reply:
Good question actually

Originally, this was made into a 0 (anything non null) because of MUI yelling at us
![image](https://github.com/bcgsc/pori_ipr_client/assets/25511396/1050ce6e-4782-4e83-b321-74eb51810eca)
And because explanations [here](https://react.dev/reference/react-dom/components/input#im-getting-an-error-a-component-is-changing-an-uncontrolled-input-to-be-controlled)

We do have null values in the DB though so I understand where you are coming from -- currently changing from either 0 or '' into null doesn't seem to affect how the API calls are being made so I'd go with default values here

Reply:
Ahh I understand now. Sounds great, I will change the default values on my end to keep it consistent with yours :)

Reply:
> changing from either 0 or '' into null doesn't seem to affect how the API calls are being made
Is the API currently being sent 0 either way?

Reply:
@elewis2 that was the question I was asking myself too when Bill posted the question, fortunately, we won't be sending the 0 if the field isn't touched -- the `isXDirty` field takes care of that, so that default 0 isn't being sent if the user doesn't specifically put 0 

Though that has the situation where the user wants to send a 0, but if the field is defaulted to 0, they won't see the 0 being updated to the backend